Makefile

```diff
@@ -30,7 +30,6 @@ $(COMMON_CONSTRAINTS_TXT):
 
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: piptools $(COMMON_CONSTRAINTS_TXT)## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
-	pip install -qr requirements/pip_tools.txt
```

Comment:
Everything makes sense, except for this bit, why remove this `pip install`?

Reply:
The `piptools` target mentioned in the above line already performs this action so this command is a duplicate one which shouldn't be needed.